### DISCUSSION TITLE
TTS in new custom language

### DISCUSSION THREAD

**Discussion by MaN0bhiR** [PaddlePaddle/PaddleSpeech#2808] [Q&A] [answered]
I have recently started using PaddleSpeech for a project. This might be naive of me , but can someone point out the pages that gives a demo of using a my own speech dataset for fine tuning a pretrained model (Acoustic) for another langugae. I have tried looking at other discussions for this , since most of them are in chinese I couldnt understand them.

**Comment by yt605155624** [comment, marked as answer]
PaddleSpeech'TTS have Chinese/English and Chinese-English-mix, if you have a new language, you **CAN NOT** use our pre-trained model, you have to train your own datasets from scratch and build your own Text Frontend
